src/components/gltf-model-plus.js

```diff
@@ -676,7 +732,7 @@ export async function loadGLTF(src, contentType, onProgress, jsonPreprocessor) {
   }
 
   return new Promise((resolve, reject) => {
-    gltfLoader.load(gltfUrl, resolve, onProgress, reject);
+    GLBRangeRequests.load(gltfUrl, gltfLoader, resolve, onProgress, reject);
```

Comment:
It's kind of odd that we have to use this static load function like this. Not sure its better than us just patching our GLTFLoader since we are bypassing the original load function anyway so we might miss upstream changes and have to think about upgrades closely anyway.

Reply:
See https://github.com/mozilla/hubs/pull/5713#discussion_r1000102783

It's a kind of a helper function. We may copy/paste the code to `gltf-model-plus`.

Reply:
I want to go as is because the range requests feature is behind the query strings now. I may revisit if it'll be enabled by default.